project/src/main/data/player-save.gd

```diff
@@ -47,13 +47,20 @@ var save_scheduled := false
 ## Provides backwards compatibility with older save formats
 var _upgrader := PlayerSaveUpgrader.new().new_save_item_upgrader()
 
+var _save_thread: Thread = null
```

Comment:
Document this field.

Reply:
Added a comment, `The currently active thread which is saving the player's data.`